src/domUpdates.js

```diff
@@ -0,0 +1,36 @@
+import dayjs from 'dayjs'
+
+//querySelectors
+const tripList = document.querySelector('.trip-list')
+const greeting = document.querySelector('.greeting')
+const annualSpending = document.querySelector('.annual-expenditure')
+
+const domUpdates = {
+    userGreeting(user) {
+        let theUser = user.getFirstName()
+        greeting.innerText = `Welcome back, ${theUser}!`
```

Comment:
Does this function need the whole user object passed in if it only uses the name?

Reply:
I see what you're saying. But I don't know how to pass in just a dynamic string. My getFirstName() doesn't doesn't exist outside of domUpdates. Does that mean I should import it here in order for that to work? I think it does, so I'll give that a try and consider this comment a "rubber duck"